src/ui/public/visualize/loader/pipeline_data_loader.ts

```diff
@@ -42,5 +43,11 @@ export class PipelineDataLoader {
         inspectorAdapters: params.inspectorAdapters,
       }
     );
+
+    if (pipelineResponse.type === 'error') {
```

Comment:
`@kbn/interpreter` doesn't actually throw errors when it catches them -- it just gives the pipeline response a `type: 'error'`, hence no `catch` here.